usgs-details.html

```diff
@@ -239,14 +292,21 @@
             xhr.send();
         }
 
+
+
```

Comment:
There are few extra lines of whitespace here, plus one on 309. We usually try to avoid committing extra whitespace, so it may be worth dropping the extra lines in a fixup commit.